compass/ocean/tests/ice_shelf_2d/viz.py

```diff
@@ -68,10 +83,81 @@ def run(self):
                                       dims=('yCell', 'nVertLevelsP1'))
         zInterface = zInterface.where(interfaceMask)
 
-        plt.figure(figsize=[24, 12], dpi=100)
-        plt.plot(ds.yCell.values, zMid.values, 'b')
-        plt.plot(ds.yCell.values, zInterface.values, 'k')
-        plt.plot(ds.yCell.values, ds.ssh.values, 'g')
-        plt.plot(ds.yCell.values, -ds.bottomDepth.values, 'g')
+        y = ds.yCell.values / 1.e3
+        plt.figure(figsize=[12, 6], dpi=100)
+        plt.plot(y, zMid.values, 'b', label='zMid')
+        plt.plot(y, zInterface.values, 'k', label='zTop')
+        plt.plot(y, ds.ssh.values, '--g', label='SSH')
+        plt.plot(y, -ds.bottomDepth.values, '--r', label='zBed')
+        plt.xlabel('Distance (km)')
+        plt.ylabel('Depth (m)')
+        plt.legend()
         plt.savefig('vert_grid.png', dpi=200)
         plt.close()
+
+        ds = xarray.open_dataset('initial_state.nc')
+
+        # Plot the time series of max velocity
+        plt.figure(figsize=[12, 6], dpi=100)
+        umax = np.amax(dsOut.velocityX[:, :, 0].values, axis=1)
+        vmax = np.amax(dsOut.velocityY[:, :, 0].values, axis=1)
+        t = dsOut.daysSinceStartOfSim.values
+        time = pd.to_timedelta(t) / 1.e9
+        plt.plot(time, umax, 'k', label='u')
+        plt.plot(time, vmax, 'b', label='v')
+        plt.xlabel('Time (s)')
+        plt.ylabel('Velocity (m/s)')
+        plt.legend()
+        plt.savefig('velocity_max_t.png', dpi=200)
+        plt.close()
+        min_column_thickness = self.config.getfloat(
+            'ice_shelf_2d', 'y1_water_column_thickness')
+
+        figsize = (4, 8)
+        delssh = dsOut.ssh - dsOut.ssh[0, :]
+        s_vmin = np.nanmin(delssh.values)
+        s_vmax = np.nanmax(delssh.values)
+        plotter.plot_horiz_series(da=delssh, nameInTitle='delssh', 
+                                  prefix='delssh', oceanDomain=True,
```

Comment:
Great! let's make the same changes to the other `plot_horiz_series` calls. Then, I also think the linter is unhappy so there's something to be figured out there.